### PR TITLE
Run DB migrations on deploy

### PR DESCRIPTION
**Pull request by MarcelGerber**
Notion: [Database migrations are not automatically run on deploy](https://www.notion.so/Database-migrations-are-not-automatically-run-on-deploy-48938a638d884cdd85efb9fd07c89c64)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "fixPrettierAll": "yarn prettier --write \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml,php}\"",
         "fixPrettierChanged": "yarn pretty-quick --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml,php}\"",
         "refreshLando": "cd wordpress && lando refresh",
+        "runDbMigrations": "yarn typeorm migration:run",
```
Comment (MarcelGerber) [inline package.json:21]: My only question for this PR:
We [used to have](https://github.com/owid/owid-grapher/blob/pre-v3/package.json#L19) `yarn knex migrate:latest && yarn typeorm migration:run` here. We don't use any Knex migrations any more, correct?

Comment (danielgavrilov) [reply package.json:21]: Yes, correct!

**Review by danielgavrilov — APPROVED** [review]
Awesome! I'm just adding some docs around this.

**Comment by danielgavrilov** [comment]
I am done – feel free to review & merge!

**Comment by MarcelGerber** [comment]
Ah, nice, I didn't notice that these were marked as deprecated.
Thank you, merging now!

**Pull request merged**